tests/lib/reset.sh

```diff
@@ -2,15 +2,13 @@
 
 # shellcheck source=tests/lib/state.sh
 . "$TESTSLIB/state.sh"
-# shellcheck source=tests/lib/systemd.sh
-. "$TESTSLIB/systemd.sh"
-
 
 reset_classic() {
     # Reload all service units as in some situations the unit might
     # have changed on the disk.
     systemctl daemon-reload
-    systemd_stop_units snapd.service snapd.socket
```

Comment:
Just want to double check it is intentional change from only stopping to stopping and removing?

Reply:
In this case it is the same because those units are regenerated when the state is restored. But I'll update that so it is more clear.
Thanks for the review.